### PR TITLE
magento/merchdocs#1122: Describe steps to allow remote shopping assistance 

### PR DESCRIPTION
**Pull request by ZhovnirMaria**
## Purpose of this pull request

This pull request describes customer steps to allow a remote shopping assistance

## Magento release version

- [x] 2.4.x

   Specify a patch release number, if applicable:

- [ ] 2.3.x

   Specify a patch release number, if applicable:

## Product editions

Is this update specific to a product edition?

- [x] Magento Open Source only
- [x] Magento Commerce only

Is this update specific to an installed feature extension

- [x] B2B extension
- [x] Other feature set, please specify:

## Affected documentation pages

https://docs.magento.com/user-guide/customers/login-as-customer.html

## Additional information

Providing Shopper Assistance | Magento 2 4 User Guide 2021-04-01 16-36-03

### CODE REVIEW

File: src/customers/login-as-customer.md [hunk 1]
```diff
@@ -21,6 +21,21 @@ _Login As Customer_
 
 Enabling Login as Customer requires that you enable the feature in your Magento instance and then enable access for Magento Admin users in the user role permissions.
 
+## Customer permission for remote shopping assistance
+
+The Magento Admin can login as a customer, only after the customer allows remote access.
```
Comment (atwixfirster) [inline src/customers/login-as-customer.md:26]: Thank you for your contributions, @ZhovnirMaria 

I would like to simplify a bit your sentence and add a preliminary explanations to steps below. Something like that:

```
To allow the ability to login as a customer for Magento admin user, a customer should make the next steps:
```

Thanks

Comment (ZhovnirMaria) [reply src/customers/login-as-customer.md]: Hi, @atwixfirster
I made some changes to the request, you can check them.
Thank you!

Comment (atwixfirster) [reply src/customers/login-as-customer.md]: Thank you, @ZhovnirMaria !

@dobooth , could you please review if the suggested changes are correct here?

Thank you!

CC: @leslietilling

File: src/customers/login-as-customer.md [hunk 2]
```diff
@@ -21,6 +21,21 @@ _Login As Customer_
 
 Enabling Login as Customer requires that you enable the feature in your Magento instance and then enable access for Magento Admin users in the user role permissions.
 
+## Customer permission for remote shopping assistance
+
+To allow the ability to login as a customer for Magento admin user, a customer should make the next steps:
```
Comment (leslietilling) [inline src/customers/login-as-customer.md:26]:
```suggestion
To allow the ability for the merchant to log in as the customer from the Admin, a customer should take these steps:
```

File: src/customers/login-as-customer.md [hunk 3]
```diff
@@ -21,6 +21,21 @@ _Login As Customer_
...
+
+1. Go to the **Account Information** page.
+
+1. Check the **Allow remote shopping assistance** checkbox.
```
Comment (leslietilling) [inline src/customers/login-as-customer.md:30]:
```suggestion
1. Select the **Allow remote shopping assistance** checkbox.
```

File: src/customers/login-as-customer.md [hunk 4]
```diff
@@ -21,6 +21,21 @@ _Login As Customer_
...
+
+1. Click the <span class="btn">Save</span> button.
+
+![]({% link images/images/permission.png %}){: .zoom}
+_Account Information Page_
+
+Without this permission, Magento Admin cannot login as this customer.
```
Comment (leslietilling) [inline src/customers/login-as-customer.md:37]:
```suggestion
Without this permission, an Admin user cannot login as this customer to provide assistance.
```

File: src/customers/login-as-customer.md [hunk 5]
```diff
@@ -21,6 +21,21 @@ _Login As Customer_
 
 Enabling Login as Customer requires that you enable the feature in your Magento instance and then enable access for Magento Admin users in the user role permissions.
 
+## Customer permission for remote shopping assistance
```
Comment (leslietilling) [inline src/customers/login-as-customer.md:24]:
```suggestion
## Customer account permission for remote shopping assistance
```

**Review by leslietilling — COMMENTED** [review]
Edits for grammar and style

**Comment by ZhovnirMaria** [comment]
Hi! @leslietilling 
I made changes, you can proceed. Thank you for your help!

**Review by leslietilling — APPROVED** [review]

**Comment by dobooth** [comment]
@magento import code to magento-devdocs/merchdocs

**Comment by magento-engcom-team** [comment]
@dobooth the branch with code successfully imported into`magento-devdocs/merchdocs` repository. Branch name: `imported-magento-merchdocs-1123`.

**Pull request merged**